maths/LargestOfVeryLargeNumbers.py

```diff
@@ -0,0 +1,29 @@
+#Author: Abhijeeth S
+
+import math                
+
+def res(x,y):                              
+    if x and y !=0:
+        return y*math.log10(x)             # We use the relation x^y = y*log10(x), where 10 is the base.
+    else:
+        if(x==0):                          # 0 raised to any number is 0
+            return 0
+        elif(y==0):
+            return 1                       # any number raised to 0 is 1
+
+# Reads two numbers from input and typecasts them to int using map function.
+# Here x is the base and y is the power.
+x1,y1 = map(int, input().split(','))
```

Comment:
Any __input()__ statements must be indented under a function or an `if __name__ == "__main__":` or our automated tests will fail.